src/library/tests/test_views.py

```diff
@@ -87,36 +131,37 @@ def test_quest_list_view(self):
         self.assertNotEqual(len(response.context['library_quests']), non_library_quest_count)
 
     def test_import_non_existing_quest_to_current_deck(self):
+        """
+        Add test that checks if the import quest to current deck view works
+        when the quest does not exist in the library schema.
+        """
+        self.client.force_login(self.test_teacher)
+        # Create a non-existing quest import_id
         url = reverse('library:import_quest', args=[str(uuid.uuid4())])
-
-        response = self.client.get(url)
-        self.assertEqual(response.status_code, 404)
+        self.assert404URL(url)
 
     def test_import_quest_already_exists_locally(self):
         """Currently we don't support overwriting existing quests (based on import_id),
         so make sure the import feature rejects already existing import_ids
         TODO: When we add an overwrite feature, this quest will need to be modified to test that feature"""
 
+        self.client.force_login(self.test_teacher)
         # Create quest in the test tenant
         quest = baker.make(Quest)
 
         # Create a quest in the libray schema with same import_id:
         with library_schema_context():
             library_quest = baker.make(Quest, import_id=quest.import_id)
 
-        url = reverse('library:import_quest', args=[library_quest.import_id])
-
-        # Test the confirmation page
-        response = self.client.get(url)
-
-        # Make a request to import the quest, should result in Permission Denied (403)
-        response = self.client.post(url)
-        self.assertEqual(response.status_code, 403)
+        # Check that the quest exitsts and sends a 200 when trying to import it
+        # It sends a 200 because there is a confirmation page that stops the import
```

Comment:
200 could mean any page, so this doesn't really definitely test anything.  It would also be 200 (in the end) if it imported the quest.  We need something more robust here I think?

Reply:
Maybe check for some text proving you hit the confirmation page? or the rejection page?

Reply:
Can you provide a screenshot here on what actually happens and what the page looks like?